src/AuthTrait.php

```diff
@@ -47,7 +47,7 @@ private function getAuthQueryAttr($attr = [])
         }
 
         if (isset($attr['scopes'])) {
-            $data['scopes'] = strval($attr['scopes']);
+            $data['scopes'] = explode(',', $attr['scopes']);
```

Comment:
Maybe is is better to have `$attr[scopes]` parameter as an array of strings?

Reply:
Probably makes sense to do that on the client side, because now we kinda force user to provide scopes separated with commas. But I was thinking about backwards compatibility, users with old versions after update will need to change source too on their side.

Reply:
That's an interesting and user-friendly approach here:) @vytautasgimbutas could make a final decision here.

Reply:
This is ok